frontend/src/Views/Export/ExportPage.tsx

```diff
@@ -13,8 +13,20 @@ import { StyledContent, StyledLandingContainer } from './ExportPage.styles.ts';
 
 const ExportPage: FC = () => {
   const downloadFile = () => {
-    localStorage.clear();
-    console.log('Downloading');
+    // localStorage.clear();
```

Comment:
is it supposed to be not cleared? Whenever I download it, and create a new calculation, the loaded  values are still there. When it should be an empty string.

Reply:
Actually, I believe it should be cleared when we click on new calculation. I'll change that, forgot to do it in the end. Nice catch!

Reply:
Had to change the link to a Button since it's now doing stuff aside from moving the page :ok_hand: